src/stream/stream.py

```diff
@@ -0,0 +1,252 @@
+"""Groundlight Stream Processor
+
+A command-line tool that captures frames from a video source and sends them to a Groundlight detector for analysis.
+Supports multiple input sources including:
+- Video devices (webcams)
+- Video files (mp4, etc)
+- RTSP streams
+- YouTube videos
+- Image directories
+- Image URLs
+"""
+
+import argparse
+import io
+import logging
+import math
+import os
+import sys
+import time
+from functools import partial
+from logging.config import dictConfig
+from queue import Queue
+
+import cv2
+import yaml
+from framegrab import MotionDetector
+from groundlight import Groundlight
+
+from stream.grabber import FrameGrabber
+from stream.image_processing import crop_frame, parse_crop_string, resize_if_needed
+from stream.threads import setup_workers
+
+fname = os.path.join(os.path.dirname(__file__), "logging.yaml")
+dictConfig(yaml.safe_load(open(fname)))
+logger = logging.getLogger(name="groundlight.stream")
+
+
```

Comment:
this file is the same as the one remove in `src/stream.py`, its just significantly pared down and so git isnt recognizing it as the same file.